scout/server/blueprints/variant/templates/variant/variant_details.html

```diff
@@ -145,43 +145,48 @@
   <div class="card panel-default">
     <div class="panel-heading d-flex justify-content-between">
       <a class="text-white" href="https://github.com/moonso/loqusdb" target="_blank">Local observations</a>
-      <span data-toggle="tooltip" title="Nr of observations is the total number of occurrences in the loqusdb instance. Missing links on greyed out case names for shared local case variants might be caused by variants not loaded after a rerun or inexact SV matching. Only names of collaborator cases are shown.">?</span>
+      <span data-toggle="tooltip" title="Nr of observations is the total number of occurrences in the loqusdb instances. Missing links on greyed out case names for shared local case variants might be caused by variants not loaded after a rerun or inexact SV matching. Only names of collaborator cases are shown.">?</span>
     </div>
     <div class="card-body">
       <table class="table">
         <thead class="thead-light">
           <tr>
+            <th scope="col">ID</th>
             <th scope="col">Nr obs.</th>
             <th scope="col">Nr homo.</th>
             <th scope="col">Total nr.</th>
           </tr>
         </thead>
         <tbody>
-          <tr>
-            <td>{{ observations.observations|default('N/A') }}</td>
-            <td>{{ observations.homozygote|default('N/A') }}</td>
-            <td>{{ observations.total }}</td>
-          </tr>
+          {% for loqusid,obs in observations.items() %}
```

Comment:
Main branch, with only one instance:

<img width="480" alt="image" src="https://user-images.githubusercontent.com/28093618/168008448-1dde2f36-5a33-4d35-b730-7d02a4c2df08.png">


This code creates something like this:

<img width="468" alt="image" src="https://user-images.githubusercontent.com/28093618/168007884-c7c7a63d-3163-4914-adf9-6bbb08fa3747.png">